tests/test_pagination.py

```diff
@@ -18,13 +18,16 @@ def name(self):
 
 
     def test_run(self):
+        # As it can call for max 100 product_variants and 
+        # we can generate only one inventory_item for one product_variants
+        excepted_streams = {'inventory_items'} 
         with self.subTest(store="store_1"):
             conn_id = self.create_connection(original_credentials=True)
-            self.pagination_test(conn_id, self.store_1_streams)
+            self.pagination_test(conn_id, self.store_1_streams-excepted_streams)
```

Comment:
Have spaces between store_1_streams and - and expected_streams
```suggestion
            self.pagination_test(conn_id, self.store_1_streams - excepted_streams)
```

Reply:
Added